src/utils/lhci-helpers.js

```diff
@@ -0,0 +1,33 @@
+const { groupBy, mapValues, orderBy } = require('lodash')
+const { join } = require('path')
+const fs = require('fs').promises
+const { existsSync } = require('fs')
+
+/**
+ * Get links by url.
```

Comment:
More context please. This gets a `links.json` file via a folder path? And returns the content of that file?